update file structure page in the node building section (#2149)
n8n-io/n8n-docs: docs/integrations/creating-nodes/build/reference/node-file-structure.md

A basic pattern is to separate out operations. Refer to the [HttpBin starter node](https://github.com/n8n-io/n8n-nodes-starter/tree/master/nodes/HttpBin){:target=_blank .external-link} for an example of this.

> For more complex nodes, n8n recommends following a directory structure like below. Refer to [Airtable node](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Airtable){:target=_blank .external-class} or [Microsoft Outlook node](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Microsoft/Outlook){:target=_blank .external-link} as examples. 

Reviewer: ```suggestion
For more complex nodes, n8n recommends a directory structure. Refer to the [Airtable node](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Airtable){:target=_blank .external-class} or [Microsoft Outlook node](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Microsoft/Outlook){:target=_blank .external-link} as examples. 
```

>   * `actions`: a directories containing sub-directories that represent resources.

Reviewer: ```suggestion
  * `actions`: a directory containing sub-directories that represent resources.
```

    * Each sub-directory should contain two types of files: 

>       * index file with resource description (named either `<resourceName>.resource.ts` or `index.ts`) 

Reviewer: ```suggestion
      * Index file with resource description (named either `<resourceName>.resource.ts` or `index.ts`) 
```
